src/services/UserService.ts

```diff
@@ -1,4 +1,6 @@
 import api from '@/api'
+import { client } from '@/api';
```

Comment:
We can all the imports from `@/api` in single line.

Reply:
Updated